estuary/api/v1.py

```diff
@@ -92,18 +92,18 @@ def get_resource_story(resource, uid):
         raise NotFound('This item does not exist')
 
     forward_query = create_story_query(item, item.id, limit=True)
-    backward_query = create_story_query(item, item.id, reverse=True, limit=True)
+    backward_query = create_story_query(item, item.id, forward=True, limit=True)
 
-    def _get_partial_story(query, reverse=False):
+    def _get_partial_story(query, forward=False):
```

Comment:
Could you please revert this parameter change back to the way it was? This is function is about getting both halves of the story and not about the sibilings.

Reply:
+1 on Matt's comment